src/actions/arcActions.ts

```diff
@@ -607,7 +628,7 @@ export function createProposal(daoAvatarAddress: string, title: string, descript
         votesYes: 0,
         votesNo: 0,
         winningVote: 0,
-        threshold: Util.fromWei(await votingMachineInstance.getThreshold({avatar: daoAvatarAddress, proposalId})).toNumber()
+        threshold: Util.fromWei(new BigNumber(await votingMachineInstance.getThreshold({avatar: daoAvatarAddress, proposalId}))).toNumber()
```

Comment:
`getThreshold` should already be returning a BigNumber

Reply:
fixed: https://github.com/daostack/alchemy/pull/81/commits/ee837c9797978fc6976021364697bca1ff0c86d0